lib/Models/SelectableDimensions.ts

```diff
@@ -0,0 +1,26 @@
+export interface DimensionOption {
```

Comment:
This file contains the interface

Reply:
Should this be a Mixin instead?

Reply:
There are instances where we've used interfaces (such as `Mappable`) and instances we've used Mixins. I think an interface works fine here.